packages/date/src/index.js

```diff
@@ -103,6 +103,11 @@ export const moment = ( ...args ) => {
 	return momentLib.tz( ...args, 'WP' );
 };
 
+// Make available other moment.js methods, such as moment.isMoment()
+Object.keys( momentLib ).forEach( ( key ) => {
```

Comment:
While I don't think we should be exporting anything at this stage, this is not the right way to expose the library either. I think it'd be better to select the APIs you want to export. Otherwise we should just be exporting a `moment` property that is the entire API. But looping through the keys and adding a reference is inefficient and also not great for documentation as it doesn't explicitly make clear the APIs we're exporting.